addOns/sequence/src/main/javahelp/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html

```diff
@@ -31,6 +34,13 @@ <H2>Scanning</H2>
 For Example:<br>
 <img src="images/ascan-sequence.png"></img>
 
+<H2>Reporting</H2>
+The following reports can include sequence scanning results:
+<ul>
+<li><a href="https://www.zaproxy.org/docs/desktop/addons/report-generation/report-traditional-json/">Traditional JSON Report</a> job.
+<li><a href="https://www.zaproxy.org/docs/desktop/addons/report-generation/report-traditional-json-plus/">Traditional JSON Report with Requests and Responses</a> job.
+</ul>
+
```

Comment:
Do we really want to maintain this is two places?

Reply:
Possibly not. Can we link to somewhere that shows this info clearly?
I dont think its too user friendly if they have to look at each report in turn to see which ones do support sequences...

Reply:
Hmmm okay. I was going to say reports help. But you're right they'd have to look around.

Report Tags? 😇

Reply:
Noooooo